terraform-modules/wfl-instance/postgres.tf

```diff
@@ -0,0 +1,18 @@
+module "postgres" {
+  source = "github.com/broadinstitute/terraform-shared.git//terraform-modules/cloudsql-postgres?ref=hf_postgres_update"
```

Comment:
Remember to replace this with a tagged version of the postgres module!

Reply:
https://github.com/broadinstitute/terraform-shared/pull/107